### ISSUE TITLE
Returning from sleep will re-enable disabled monitor

### ISSUE THREAD

**Issue by and-rea** [Eun/DisableMonitor#56]
Hello, I have this current setup:
- MacOS 10.11.5
- iMac (21.5-inch, Late 2009) monitor disabled
- secondary monitor (BenQ GW2765) attached by DisplayPort,  enabled and set as primary

When monitor returns from sleep, the iMac monitor is no longer disabled, and I need to disable it again manually.

**Comment by reagle** [comment]
Yes, this significantly decreases its utility as my monitor sleeps often.
